.github/workflows/test-integrations.yml

```diff
@@ -81,7 +81,6 @@ jobs:
     strategy:
       matrix:
         nomad-version: ['v1.6.2', 'v1.5.9', 'v1.4.13']
-
```

Comment:
~ Maybe unintentional line removal here, though looks like it'll sync to Ent cleanly

Reply:
yep, certainly not intentional.. :thinking: I think the kitten might have helped out with that one when it was walking on my keyboard.

Reply:
Interesting.. I looked into this further and it seems that https://github.com/hashicorp/consul/commit/61bd08c8b9a16cb417400e102b14c95fb504cab7 introduced the extra newline on the `main` branch but this whitespace change wasn't ported to the `release/*` branch(es).  What do you think about deleting this extra newline so this file is consistent with the release branches. Maybe this is a bit pedantic :shrug:

Reply:
I like it 👍🏻 better to be consistent, and needs no further work. Thanks for figuring that out!